hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerWithMultiWriter.java

```diff
@@ -404,12 +405,24 @@ private void runJobsInParallel(String tableBasePath, HoodieTableType tableType,
        * Need to perform getMessage().contains since the exception coming
        * from {@link org.apache.hudi.utilities.deltastreamer.HoodieDeltaStreamer.DeltaSyncService} gets wrapped many times into RuntimeExceptions.
        */
-      if (expectConflict && e.getCause().getMessage().contains(ConcurrentModificationException.class.getName())) {
+      if (expectConflict && backfillFailed.get() && e.getCause().getMessage().contains(ConcurrentModificationException.class.getName())) {
         // expected ConcurrentModificationException since ingestion & backfill will have overlapping writes
-        if (backfillFailed.get()) {
+        if (!continuousFailed.get()) {
```

Comment:
NTR: in most cases backfill job fails and hence the test succeeds. but if continuous job fails, the test times out.